tests/route/test_route_flow_counter.py

```diff
@@ -10,7 +10,7 @@
 allure.logger = logger
 
 pytestmark = [
-    pytest.mark.topology("any")
+    pytest.mark.topology('t0', 't1', 'm0', 'mx'),
```

Comment:
better to skip for t2 specifically as "any" will include all topologies like dual tor or t1-lag etc

Reply:
isntead of making this change can we update api `def get_route_flow_counter_capability(dut, route_flow_counter_capability):` return False . We can check either multi-asic or top as t2

Reply:
@abdosi don't think we should return False if T2/MA, if so the test will always fail for T2

```
    if not wait_until(CAPABILITY_WAIT_TIME_IN_SEC, CAPABILITY_CHECK_INTERVAL_IN_SEC, 0,
                      get_route_flow_counter_capability, rand_selected_dut, route_flow_counter_capability):
        pytest_assert(False, 'Failed to get route flow counter capability')
```